pages/about.js

```diff
@@ -25,46 +35,87 @@ const Mission = () => (
   </ContentSection>
 );
 
-const ContributorsList = contributorsData.map((person, index) => {
-  if (!person.login) {
-    return null;
-  }
-  return (
-    <div key={index} className="person">
-      <div className="person-avatar">
-        <img src={person.avatar_url} />
-      </div>
-      <div className="person-details">
-        <div className="person-name">{person.name || "@" + person.login}</div>
-      </div>
-      <div className="person-social">
-        <div key={index} className="social-item">
-          <a href={person.html_url} target="_blank">
-            <i className="mdi mdi-github-circle" />
+const Contributor = ({ person }: { person: Member }) => (
+  <div className="person">
+    <div className="person-avatar">
+      <img src={person.avatarUrl} />
+    </div>
+    <div className="person-details">
+      <div className="person-name">{person.name || "@" + person.login}</div>
+    </div>
+    <div className="person-social">
+      <div className="social-item">
+        <a href={person.url} target="_blank">
+          <i className="mdi mdi-github-circle" />
+        </a>
+        {person.websiteUrl ? (
+          <a href={person.websiteUrl} target="_blank">
+            <i className="mdi mdi-web" />
           </a>
-          {person.blog ? (
-            <a href={person.blog} target="_blank">
-              <i className="mdi mdi-web" />
-            </a>
-          ) : null}
-        </div>
+        ) : null}
       </div>
     </div>
+  </div>
+);
+
+const Contributors = ({ members }: { members: ?Members }) => {
+  if (!members || members.length === 0) return null;
+  return (
+    <ContentSection>
+      <ContentSectionPane layout="center">
+        <h3>Contributors</h3>
+        <div className="grid">
+          <div className="grid-wrapper">
+            {members.map(person => (
+              <Contributor person={person} key={person.login} />
+            ))}
+          </div>
+        </div>
+      </ContentSectionPane>
+    </ContentSection>
   );
-});
+};
 
-const Contributors = () => (
-  <ContentSection>
-    <ContentSectionPane layout="center">
-      <h3>Contributors</h3>
-      <div className="grid">
-        <div className="grid-wrapper">{ContributorsList}</div>
-      </div>
-    </ContentSectionPane>
-  </ContentSection>
-);
+export default class AboutPage extends Component<void, OSProps, void> {
+  static async getInitialProps() {
+    const client = new GraphQLClient("https://api.github.com/graphql", {
+      headers: {
+        Authorization: "Bearer 243492ac50267b053ca0d23d73a59f9e46bee565"
```

Comment:
This is my personal token, though it doesn't have any privileges apart from the public ones enabled. So I guess it's OK to share it public.

Reply:
😱 

We can make this an environment variable and make it part of the deployment.

Reply:
Do you know how we can do this with now?